__tests__/UsingTypeScript.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { StrictMode, ChangeEvent, forwardRef } from 'react';
+import { view as v, store as s } from 'react-easy-state';
+import { render, cleanup } from '@testing-library/react/pure';
+
+const store = s({
+  name: 'Peter',
+  setNameEvent(ev: ChangeEvent<HTMLInputElement>) {
+    if (ev.target) {
+      this.name = ev.target.value;
+    }
+  },
+});
+
+// properly retain these props for the views
+type AppProps = {
+  greeting: string;
+};
+
+const App = v(
+  forwardRef<HTMLLabelElement, AppProps>(
+    ({ greeting, children }, ref) => (
+      <div>
+        <h1>
+          {greeting} {store.name}!
+        </h1>
+        <label ref={ref} htmlFor="name">
+          Name:
+        </label>
+        <input
+          id="name"
+          value={store.name}
+          onChange={store.setNameEvent.bind(store)}
+        />
+        {children}
+      </div>
+    ),
+  ),
+);
+
+describe('Using TypeScript App', () => {
+  afterAll(() => {
+    cleanup();
+  });
```

Comment:
I think this is not going to work if you rebase to master and run `npm i`.
Check this README: https://github.com/RisingStack/react-easy-state/blob/master/__tests__/README.md
You will have to change this line to `afterAll(cleanup)`

Reply:
And I've just realized that the README is not 100% clear. You cannot even run the `cleanup` in a function as you did.

Reply:
Something got messed up at `@testing-library/dom` which causing this issue.